determined-ai/determined
# feat: move task logs to embedded view [DET-7169]

## Description

feat: move task logs to embedded view [DET-7169]

### Updates
- Clicking "View Logs" from the context menu on the interactive task page now opens up the log view within the interactive task page with the task bar still visible.
- The embedded Task Logs view has an additional `X` button in the top right corner that will close the logs and re-open the interactive task page. 
- The original Task Logs pages remains unchanged. 




## Test Plan

* Create a JupyterLab Notebook or a Tensorboard session
    * Click `View Logs` from the context menu and ensure that you the logs for the task open up in the same page.
    * Click the `X` in the top right hand corner and ensure that the page re-opens the task
    * Click `View Logs` from the `Tasks` page for the task and ensure that you are shown the logs in the normal log viewing page and that no `X` button is present.



## Commentary (optional)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-7169]: https://determinedai.atlassian.net/browse/DET-7169?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
